### DISCUSSION TITLE
WBO bond orders

### DISCUSSION THREAD

**Discussion by woodg07** [grimme-lab/xtb-python#46] [Ideas]
Really like what is happening with this project.  Looking at the documentation it seems that the number of properties being exposed is currently only a handful of methods.  Is there any way to request the wbo method to be added as a property in upcoming releases?

**Comment by awvwgk** [comment]
I think I have support for bond orders implemented already:
https://github.com/grimme-lab/xtb-python/blob/bc55eeb97194e0c39f997e74c07eea095dc7b6f6/xtb/interface.py#L505

**Reply by woodg07** [reply]
That's great thank you. For some reason I was trying to fish out the results from here:
https://xtb-python.readthedocs.io/en/latest/qcarchive.html

so for example how do you set _wfn so it returns the mayer indices in that case?

**Reply by awvwgk** [reply]
The QCSchema doesn't really schematize bond orders, but there is an inofficial schema by Psi4 providing the bond orders under the name `mayer_indices` when requesting properties as result from QCEngine. In all other cases the bond orders will be stored under `["extras"]["xtb"]["mayer_indices"]` in the returned atomic results object as well. So you can certainly get bond orders when using `xtb` with QCEngine.
